src/components/viewer.ts

```diff
@@ -230,6 +230,7 @@ export class Viewer {
                             spreadMode: configuration.get('view.pdf.spreadMode'),
                             hand: configuration.get('view.pdf.hand'),
                             invert: configuration.get('view.pdf.invert'),
+                            keybinding: configuration.get('view.pdf.internal.synctex.keybinding')
```

Comment:
To make other keybindings configurable, please do not set the value of the keybinding of synctex directly.

I prefer:
```typescript
 keybinding: { synctex: configuration.get('view.pdf.internal.synctex.keybinding') }
```